scale-tests/run.sh

```diff
@@ -319,15 +365,18 @@ else
   log 'Skipping infrastructure installation'
 fi
 
+################################################################################
+# Install non-GPU Spark dispatchers ############################################
+################################################################################
+
 if [ "${SHOULD_INSTALL_NON_GPU_DISPATCHERS}" = true ]; then
   log 'Installing non-GPU dispatchers'
   start_time=$(date +%s)
   container_exec \
     ./scale-tests/deploy-dispatchers.py \
-      --quota-drivers-cpus "${NON_GPU_QUOTA_DRIVERS_CPUS}" \
-      --quota-drivers-mem "${NON_GPU_QUOTA_DRIVERS_MEM}" \
-      --quota-executors-cpus "${NON_GPU_QUOTA_EXECUTORS_CPUS}" \
-      --quota-executors-mem "${NON_GPU_QUOTA_EXECUTORS_MEM}" \
+      --group-role "${ROLE_NAME}" \
+      --options-json "${SPARK_CONFIG}" \
+      --create-quotas false \
```

Comment:
Assuming you will create a separate PR to create quota the new way?

Reply:
The quota creation automation was added in https://github.com/mesosphere/spark-build/pull/550/commits/d238830152f74aa1fb94af1ab9c4eb91fb33dfa5#diff-5f81cedb7de857f3a6139a9d6ed6d81bR321-R323.

Reply:
I misunderstood your question in my reply. Here is the automatic quota creation: https://github.com/mesosphere/spark-build/pull/550/commits/a4c89862606637731cf9f5f3e043980ded96ebc0